frontend/src/components/ConfirmationDialog.scss

```diff
@@ -0,0 +1,94 @@
+/* ConfirmationDialog.css */
+.modal-overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: rgba(0, 0, 0, 0.5);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  z-index: 1000;
+}
+
+.modal-content {
+  background-color: white !important;
+  border-radius: 8px;
+  width: 100%;
+  max-width: 60vw;
+  padding: 24px;
+  box-shadow: 0 2px 10px rgba(0, 0, 0, 0.1);
+  position: relative;
+}
+
+.modal-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+}
+
+.modal-header h2 {
+  margin: 0;
+  font-size: 20px;
+  color: #333;
+}
+
+.close-button {
+  background: none;
+  border: none;
+  font-size: 24px;
+  color: #666;
+  cursor: pointer;
+  padding: 4px;
+  line-height: 1;
+}
+
+.close-button:hover {
+  color: #333;
+}
+
+.modal-body {
+  margin-bottom: 24px;
+}
+
+.modal-body p {
+  margin: 0;
+  color: #666;
+}
+
+.modal-footer {
+  display: flex;
+  justify-content: flex-end;
+  gap: 12px;
+}
+
+.button {
+  padding: 8px 16px;
+  border-radius: 4px;
+  font-size: 14px;
+  cursor: pointer;
+  border: none;
+  font-weight: 500;
+  transition: background-color 0.2s;
+}
+
+.button-secondary {
+  background-color: white;
+  border: 1px solid #ddd;
+  color: #333;
+}
+
+.button-secondary:hover {
+  background-color: #f5f5f5;
+}
+
+.button-primary {
+  background-color: #1a365d;
+  color: white;
+}
+
+.button-primary:hover {
+  background-color: #152a4a;
+}
```

Comment:
We shouldn't need any of these styles since we've already got Bootstrap providing them with "btn"
https://getbootstrap.com/docs/5.3/components/buttons/